apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -268,6 +268,7 @@ def _load_skill(path, course):
 
     return Skill(
         name=name,
+        filename=str(path)[len(course.course_dir)+1:],
```

Comment:
why `str(path)` instead of just `path`?

Reply:
btw if I understand correctly, this should be done using [os.path.relpath](https://docs.python.org/3/library/os.path.html#os.path.relpath):

```suggestion
        filename=os.path.relpath(path, start=course.course_dir),
```

Reply:
yes, that's much better